php-classes/Slate/CBL/Competency.php

```diff
@@ -271,8 +288,9 @@ public function getCompletionForStudent(Student $Student, $level = null, $demons
         return [
             'demonstrationsLogged' => 0,
             'demonstrationsComplete' => 0,
+            'baselineRating' => $StudentCompetency && $StudentCompetency->BaselineRating ? floatval($StudentCompetency->BaselineRating) : null,
```

Comment:
@nbey this can just be `$StudentCompetency ? $StudentCompetency->BaselineRating : null` once the ActiveRecord returns the correct type on its own